pages/common/meteor.md

```diff
@@ -0,0 +1,27 @@
+# meteor
+
+> Full-stack javascript platform for building web applications.
+
+- Run a meteor project from its root directory in development mode:
+
+`meteor`
+
+- Create a project under the given directory:
+
+`meteor create {{path/to/directory}}`
+
+- Display the list of packages your project is currently using:
+
+`meteor list`
+
+- Add a package to your project:
+
+`meteor add {{package_name}}`
+
+- Remove a package from your project:
+
+`meteor remove {{package_name}}`
+
+- Create a production build of your project under the given directory. (Omit the --directory option if you want to wrap the build inside a tarball instead):
```

Comment:
idem (the second "your" could be removed with simply "if you want to" --> "to").

Also, the second part could be a separate example. It would make this page longer than we've been comfortable with so far, but since we haven't made a decision yet in #686, I can't really enforce it, just discourage it.